src/elements/types/ExportSpecifier.js

```diff
@@ -0,0 +1,43 @@
+import ModuleSpecifier from '../ModuleSpecifier';
+
+// tests are in ExportNamedDeclaration
+
+export default class ExportSpecifier extends ModuleSpecifier {
+    constructor(childNodes) {
+        super('ExportSpecifier', childNodes);
+    }
+
+    _acceptChildren(children) {
+        console.log(children._elements.map(function(e) {
+            return e._type;
+        }));
+
+        let local;
+        console.log(children._currentElement);
+        console.log(children.isNode('Identifier'));
+        console.log(children.isToken('Identifier'));
```

Comment:
The confusion comes from the fact that there is a node `Identifier` and at the same time there is also a token `Identifier` (which is ofter inside node `Identifier`).

For instance of you have `x = 1`, left side of the expressions is Node `Identifier` with Token `Identifier` inside. But if you have `super()`, then you have a Node `Super` with Token `Identifier` inside.